plot_fit.py

```diff
@@ -78,6 +78,12 @@ def plot_fit(bestfits_source, geometries_selection,
              alpha_allmodels=0.1,
             ):
 
+    # This code will break if you don't have the following variables defined and passed into the function:
+    # fieldid = 'G328' (ex. - whatever your region is)
+    # spicyid = 31415 (ex. - whatever source you're looking at)
+    # name = 'btingle' (however your name appears in your directory, aka /home/yourname)
+    # They are used to pull the location image.
+    
```

Comment:
these are called "positional arguments".

The standard is to document them as you have, but in a "docstring", e.g., like this:
```suggestion
    """
    Parameters
    ----------
    fieldid : string
         'G328' (ex. - whatever your region is)
    """
```
etc.